fastpin.h

```diff
@@ -206,14 +206,14 @@ template<uint8_t PIN> class FastPin {
 
 	static_assert(validpin(), "Invalid pin specified");
 
-	static void _init() {
-	}
+	static void _init() { }
 public:
 	typedef volatile RwReg * port_ptr_t;
 	typedef RwReg port_t;
 
 	inline static void setOutput() { }
 	inline static void setInput() { }
+    inline static void setInputPullup() { }
```

Comment:
This functionality should probably get propagated to all the pin classes across platforms before merging this PR in.